functions/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import * as functions from 'firebase-functions';
+import * as admin from "firebase-admin";
+import {onCreateFunction1, onCreateFunction2} from "./notifications";
```

Comment:
Man kan lägga in själva "lyssnarna" i den andra filen också om man vill ha indexfilen extra clean. Kolla in [denna](ttps://firebase.google.com/docs/functions/organize-functions#group_functions) sida för inspo.